src/index.tsx

```diff
@@ -8,7 +8,7 @@ import { rootReducer } from "./reducers";
 // css import
 import "antd/dist/antd.css";
 
-const store = createStore(rootReducer);
+const store: any = createStore(rootReducer);
```

Comment:
Just out of curiosity: was there any error from the linter? Because it seemed to work without it as well.